src/platform/PICO/io_def_generated.h

```diff
@@ -0,0 +1,230 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// DEFIO_PORT_<port>_USED_MASK is bitmask of used pins on target
+// DEFIO_PORT_<port>_USED_COUNT is count of used pins on target
+
+#if defined(RP2350A)
+#define DEFIO_PORT_A_USED_COUNT 30
+#elif defined(RP2350B)
+#define DEFIO_PORT_A_USED_COUNT 48
+#else
+#error "Unsupported target MCU type for PICO"
+#endif
+
+#define DEFIO_PORT_A_OFFSET (0)
+
+// DEFIO_GPIOID__<port> maps to port index
+#define DEFIO_GPIOID__A 0
+
+// DEFIO_TAG__P<port><pin> will expand to TAG if defined for target, error is triggered otherwise
+// DEFIO_TAG_E__P<port><pin> will expand to TAG if defined, to NONE otherwise (usefull for tables that are CPU-specific)
+// DEFIO_REC__P<port><pin> will expand to ioRec* (using DEFIO_REC_INDEX(idx))
+
+#define DEFIO_TAG__PA0    DEFIO_TAG_MAKE(DEFIO_GPIOID__A, 0)
```

Comment:
This should match PICO naming conventions (probably P0-P47, possibly named pins from bank 1), there is port 'A' on PICO.

Reply:
There no concept of port in the PICO, it named pin 1..n in future versions this could be 64, 128 etc currently limited to 48.

There is however low and high registers to get around the 32bit limit.